gobblin-core/src/main/java/gobblin/source/extractor/extract/sftp/SftpFsHelper.java

```diff
@@ -43,7 +47,7 @@
  */
 public class SftpFsHelper implements SizeAwareFileBasedHelper {
```

Comment:
I think the name is misleading. Typically, I expect a help class to be a set of static methods. This class does session management so I think it should be named SftpSession.